### ISSUE TITLE
Removing break point has no effect until session is restarted

### ISSUE THREAD

**Issue by bilogic** [xdebug/vscode-php-debug#380]
Adapter version: 1.13.0
PHP 7.2.10 (cli) (built: Sep 11 2018 09:59:29) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.10, Copyright (c) 1999-2018, by Zend Technologies
    with Xdebug v2.6.1, Copyright (c) 2002-2018, by Derick Rethans

1. Debugging currently works, i.e. break points are triggered, variable values can be read etc
2. If I try to remove an existing break point on line X, the red dot in VSCode visually disappears
3. But the debugger still stops when reaching line X, i.e. the break point is actually not removed
4. The break point is only truly removed when I restart the session
5. However, during a session, I'm able to add break points

Please help, thank you!

**Comment by bilogic** [comment]
I managed to capture some logs, and I think it shows the issue

1. Enable breakpoint on sslcert.php, line 232
```
<- breakpoint_list -i 155
-> <response xmlns="urn:debugger_protocol_v1" xmlns:xdebug="http://xdebug.org/dbgp/xdebug" command="breakpoint_list" transaction_id="155"><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830008"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="144" state="enabled" hit_count="0" hit_value="0" id="107830009"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830028"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830029"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830048"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830049"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830068"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830069"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830022"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830044"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830066"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830001"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830023"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830045"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830067"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830002"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830020"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830046"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830064"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830003"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830021"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830047"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830065"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="144" state="enabled" hit_count="0" hit_value="0" id="107830004"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830026"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830040"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830062"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830005"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830027"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830041"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830063"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830006"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830024"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830042"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830060"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830007"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830025"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830043"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830061"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830019"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830018"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830039"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830038"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830059"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830058"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830011"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830033"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830055"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830010"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830032"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830054"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830013"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830031"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830057"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830075"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830012"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830030"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830056"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830074"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830015"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830037"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830051"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830073"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="144" state="enabled" hit_count="0" hit_value="0" id="107830014"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830036"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830050"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830072"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830017"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830035"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830053"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830071"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830016"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830034"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830052"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830070"></breakpoint></response>

<- breakpoint_set -i 156 -t line -f file:///sslcert.php -n 173
-> <response xmlns="urn:debugger_protocol_v1" xmlns:xdebug="http://xdebug.org/dbgp/xdebug" command="breakpoint_set" transaction_id="156" id="107830076"></response>

<- breakpoint_set -i 157 -t line -f file:///sslcert.php -n 178
-> <response xmlns="urn:debugger_protocol_v1" xmlns:xdebug="http://xdebug.org/dbgp/xdebug" command="breakpoint_set" transaction_id="157" id="107830077"></response>

<- breakpoint_set -i 158 -t line -f file:///sslcert.php -n 232
-> <response xmlns="urn:debugger_protocol_v1" xmlns:xdebug="http://xdebug.org/dbgp/xdebug" command="breakpoint_set" transaction_id="158" id="107830078"></response>
```

2. Disable breakpoint on sslcert.php, line 232
```
<- breakpoint_list -i 159
-> <response xmlns="urn:debugger_protocol_v1" xmlns:xdebug="http://xdebug.org/dbgp/xdebug" command="breakpoint_list" transaction_id="159"><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830008"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="144" state="enabled" hit_count="0" hit_value="0" id="107830009"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830028"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830029"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830048"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830049"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830068"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830069"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830022"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830044"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830066"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830001"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830023"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830045"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830067"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830002"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830020"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830046"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830064"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830003"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830021"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830047"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830065"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="144" state="enabled" hit_count="0" hit_value="0" id="107830004"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830026"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830040"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830062"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830005"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830027"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830041"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830063"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830006"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830024"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830042"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830060"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830007"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830025"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830043"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830061"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830019"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830018"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830039"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830038"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830059"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830058"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830078"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830011"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830033"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830055"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830077"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830010"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830032"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830054"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830076"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830013"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830031"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830057"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830075"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830012"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830030"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830056"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830074"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830015"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830037"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830051"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830073"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="144" state="enabled" hit_count="0" hit_value="0" id="107830014"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830036"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830050"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830072"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830017"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830035"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830053"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="173" state="enabled" hit_count="2" hit_value="0" id="107830071"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="142" state="enabled" hit_count="1" hit_value="0" id="107830016"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830034"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="178" state="enabled" hit_count="1" hit_value="0" id="107830052"></breakpoint><breakpoint type="line" filename="file:///sslcert.php" lineno="232" state="enabled" hit_count="0" hit_value="0" id="107830070"></breakpoint></response>

<- breakpoint_set -i 160 -t line -f file:///sslcert.php -n 178
-> <response xmlns="urn:debugger_protocol_v1" xmlns:xdebug="http://xdebug.org/dbgp/xdebug" command="breakpoint_set" transaction_id="160" id="107830079"></response>

<- breakpoint_set -i 161 -t line -f file:///sslcert.php -n 232
-> <response xmlns="urn:debugger_protocol_v1" xmlns:xdebug="http://xdebug.org/dbgp/xdebug" command="breakpoint_set" transaction_id="161" id="107830080"></response>
```
Even after disabling the breakpoint in 2, ```breakpoint_list``` grows longer and longer, why is that?

**Comment by bilogic** [comment]
To anyone else facing this issue, best to switch to https://github.com/RobberPhex/vscode-php-debug version 1.14.0 who has fixed the bug

**Comment by zobo** [comment]
This was addressed in the breakpoint handling rewrite #550

**Comment by JVT038** [comment]
This issue still happens to me in version 1.34.0
